src/main/java/lottery/domain/ShufflePicker.java

```diff
@@ -0,0 +1,22 @@
+package lottery.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ShufflePicker implements Picker {
+    private final List<LotteryNumber> basis;
+
+    public ShufflePicker() {
+        basis = IntStream.rangeClosed(LotteryNumber.MIN, LotteryNumber.MAX)
+                .mapToObj(LotteryNumber::new)
+                .collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private static final List<LotteryNumber> BASIS =
        IntStream.rangeClosed(LotteryNumber.MIN, LotteryNumber.MAX)
                .mapToObj(LotteryNumber::new)
                .collect(Collectors.toList());
```
static 변수로 관리해도 되지 않았을까요?